3rd-party/woocommerce-stats.php

```diff
@@ -0,0 +1,139 @@
+<?php
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+class WC_Stats {
+	/**
+	 * @var Jetpack
+	 **/
+	private $jetpack;
+
+	/**
+	 * @var WC_Stats
+	 **/
+	private static $instance = null;
+
+	static function init() {
+		// Tracking only Site pages
+		if ( is_admin() ) {
+			return;
+		}
+		// Make sure Jetpack is installed and active
+		if ( ! Jetpack::is_active() ) {
+			return;
+		}
+		// Make sure WooCommerce is installed and active
+		if ( ! in_array( 'woocommerce/woocommerce.php', apply_filters( 'active_plugins', get_option( 'active_plugins' ) ) ) ) {
+			return;
+		}
+		if ( is_null( self::$instance ) ) {
+			self::$instance = new WC_Stats();
+		}
+		return self::$instance;
+	}
+
+	public function __construct() {
+		$this->jetpack = Jetpack::init();
+
+		add_action( 'wp_enqueue_scripts', array( $this, 'track' ) );
+		add_action( 'rest_api_init', function() {
```

Comment:
We can't use anonymous functions in Jetpack yet, since we have to be compatible with PHP 5.2